src/main/java/com/acrolinx/sidebar/utils/SidebarUtils.java

```diff
@@ -141,30 +139,24 @@ public static String getSidebarUrl(final String serverAddress) {
   }
 
   private static String getCurrentSdkImplementationVersion() {
-    final Optional<String> versionFromManifestFile = getJavaSdkVersionFromManifestFile();
+    final Optional<String> versionFromManifestFile = getJavaSdkVersionFromPropertiesFile();
 
     return versionFromManifestFile.orElseGet(
         () -> SidebarUtils.class.getPackage().getImplementationVersion());
   }
 
-  private static Optional<String> getJavaSdkVersionFromManifestFile() {
+  private static Optional<String> getJavaSdkVersionFromPropertiesFile() {
     final ClassLoader classLoader = SidebarUtils.class.getClassLoader();
 
     try {
-      final Enumeration<URL> enumeration = classLoader.getResources("META-INF/MANIFEST.MF");
+      // Load the properties file from the JAR
+      InputStream inputStream =
+          classLoader.getResourceAsStream("sidebar-sdk-java-version.properties");
+      Properties properties = new Properties();
+      properties.load(inputStream);
 
-      while (enumeration.hasMoreElements()) {
-        final URL url = enumeration.nextElement();
+      return Optional.ofNullable(properties.getProperty("version"));
```

Comment:
maybe a better property name would be "sidebar-sdk-java.version" or so, just to be extra clear what this is?